packages/driver/cypress/integration/e2e/multi-domain/snapshots/multi_domain_snapshot_files.spec.ts

```diff
@@ -0,0 +1,67 @@
+import { findCrossOriginLogs } from '../../../../support/utils'
+
+// @ts-ignore / session support is needed for visiting about:blank between tests
+context('cross-origin snapshot files', { experimentalSessionSupport: true }, () => {
+  let logs: Map<string, any>
+
+  beforeEach(() => {
+    logs = new Map()
+
+    cy.on('log:changed', (attrs, log) => {
+      logs.set(attrs.id, log)
+    })
+
+    cy.visit('/fixtures/multi-domain.html')
+    cy.get('a[data-cy="cross-origin-secondary-link"]').click()
+  })
+
+  // NOTE: there is no command log for .fixture(). shows up as a (new url)
+  it.skip('.fixture()', () => undefined)
```

Comment:
This applies to some other places, too.

Reply:
We do not. I figured having the command basis covered so that others qho look at this code would understand why there isn't a test for some commands. But that might be very well implied if we move these tests into the others with a consoleProps sections.

Reply:
updated in [9734059](https://github.com/cypress-io/cypress/pull/20949/commits/9734059077f187f410d9163a8922c5359fef4024)